vars/sonarqubeStaticAnalysis.txt

```diff
@@ -0,0 +1,35 @@
+# SonarQube Static Analysis
+
+This function will validate that a build server webhook has been configured in 
+SonarQube, run the sonar:sonar maven goal for the configured pom file, and then 
+wait for the quality gate status to be OK.  The pipeline will be stopped if the
+webhook has not been configured or the quality gate fails.
+
```

Comment:
Perhaps it should be mentioned that  https://github.com/redhat-cop/containers-quickstarts/tree/master/sonarqube creates the web hook for you?